src/WaveTrack.h

```diff
@@ -609,6 +610,8 @@ class AUDACITY_DLL_API WaveTrack final : public PlayableTrack {
    // Protected methods
    //
 
+   void PasteWaveTrack(double t0, const WaveTrack* other);
```

Comment:
I notice that this is a protected method.  I notice also that WaveTrack is a final class, so the distinction of protected and private is not useful.  Maybe the class should be simplified, only to have public and private methods.